.github/workflows/release.yml

```diff
@@ -37,15 +44,26 @@ jobs:
         with:
           node-version: "14"
           cache-dependency-path: "ui/yarn.lock"
+
       - name: Install Yarn
         run: |
           npm install -g yarn
-      - uses: actions/checkout@v2
+
+      - name: Update version file
+        run: |
+          NOMAD_MAIN_VERSION = $(echo "${{ inputs.version }}" | cut -d- -f1)
+          NOMAD_PRERELEASE_VERSION = $(echo "${{ inputs.version }}" | sed 's|^[^-]*-\{0,1\}||g' )
+          echo "updating version to $(NOMAD_MAIN_VERSION)-$(NOMAD_PRERELEASE_VERSION)"
+          sed -i.bak -e 's|\(Version * = *"\)[^"]*|\1$(NOMAD_MAIN_VERSION)|g' version/version.go
+          sed -i.bak -e 's|\(VersionPrerelease * = *"\)[^"]*|\1$(NOMAD_PRERELEASE_VERSION)|g' version/version.go
+          rm -rf version/version.go.bak
```

Comment:
Just realized that we missed this from the original release:
https://github.com/hashicorp/nomad/blob/main/scripts/release/Makefile.linux#L4-L11

Reply:
We'd eventually like this to be [kept up to date automatically](https://hashicorp.atlassian.net/browse/RDX-196?atlOrigin=eyJpIjoiMzhhODYxOTA4YmM5NGRjNDlmYzc2MzRjZmMwZDA5OTUiLCJwIjoiaiJ9) on the release branches - but I think this is fine for now since you are committing the changed files anyway.

Reply:
Yeah, it will be great to have that, but for now I'm trying to minimize steps for the release. This is what we've been doing, and it has worked so far, so we'll keep it until that item is done.

Reply:
Yes, absolutely!